xunit.runner.wpf/MainWindow.xaml.cs

```diff
@@ -11,12 +11,15 @@ public partial class MainWindow : Window
     {
         public static Window Instance { get; private set; }
 
+        // WPF generates fields that are marked as non-nullable, but not definitely initialized.
+#pragma warning disable CS8618
```

Comment:
That's going to be a problem if we have to do this for every single WPF control.